extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerConfiguration.java

```diff
@@ -19,8 +16,92 @@
     public String nodeName;
 
     /**
-     * The default transaction timeout
+     * The default transaction timeout in seconds
      */
-    @ConfigItem(defaultValue = "60")
+    @ConfigItem(defaultValue = "300")
     public Optional<Duration> defaultTransactionTimeout;
+
+    /**
+     * The directory name of location of the transaction logs.
+     * If the value is not absolute then the directory is relative
+     * to the <em>user.dir</em> system property.
+     */
+    @ConfigItem(name = "objectStoreDir", defaultValue = "ObjectStore")
```

Comment:
The `name` option should not be given here; the property would be called `object-store-dir`.

Reply:
It was my intention to call the property `object-store-dir` since that is the quarkus naming convention. Please can you elaborate.

Reply:
Giving the `name` property sets the literal property name to that string, so in configuration, this property would be called `objectStoreDir` (which is the name given in the example configuration in a different part of the PR).

The default value for the `name` property of `ConfigItem` is `ConfigItem#HYPHENATED_ELEMENT_NAME`, which tells the config parser system to de-camel-case and hyphenate the field name and use that as the property name.  By giving an explicit `name` property on this annotation, you've defeated that mechanism and overriden the name which would have been `object-store-dir`, causing it to be `objectStoreDir` instead, which does not follow the naming convention.

Reply:
Ah I think I understand. So if no name is given then quarkus will automatically select the name `object-store-dir`. That makes sense.

Reply:
and your comment would also apply to the property `checkForNarayanaSystemProperties`. Thanks.